packages/grafana-ui/src/components/TimePicker/TimePicker.tsx

```diff
@@ -11,24 +11,27 @@ import { TimePickerPopover } from './TimePickerPopover';
 import { ClickOutsideWrapper } from '../ClickOutsideWrapper/ClickOutsideWrapper';
 
 // Utils & Services
-import { isDateTime, DateTime } from '@grafana/data';
-import { rangeUtil } from '@grafana/data';
+import { isDateTime, DateTime, rangeUtil } from '@grafana/data';
 import { rawToTimeRange } from './time';
+import { stylesFactory } from '../../themes/stylesFactory';
 import { withTheme } from '../../themes/ThemeContext';
 
 // Types
-import { TimeRange, TimeOption, TimeZone, TIME_FORMAT, SelectableValue, dateMath } from '@grafana/data';
-import { GrafanaTheme } from '@grafana/data';
+import { TimeRange, TimeOption, TimeZone, TIME_FORMAT, SelectableValue, dateMath, GrafanaTheme } from '@grafana/data';
 import { Themeable } from '../../types';
 
-const getStyles = memoizeOne((theme: GrafanaTheme) => {
+const getStyles = stylesFactory((theme: GrafanaTheme) => {
+  const yellowLight = tinycolor(theme.colors.yellow)
+    .lighten(20)
+    .toString();
   return {
     timePickerSynced: css`
       label: timePickerSynced;
       border-color: ${theme.colors.orangeDark};
       background-image: none;
       background-color: transparent;
       color: ${theme.colors.orangeDark};
+      box-shadow: 0px 0px 4px ${yellowLight};
```

Comment:
Add glow according to [design](https://www.figma.com/file/I34UgAga9JvqjlxfKYZnve/Components?node-id=453%3A0).